resources/views/site/blocks/showcase.blade.php

```diff
@@ -8,59 +9,137 @@
     $tag = $block->input('tag');
     $linkLabel = $block->input('link_label');
     $linkUrl = $block->input('link_url');
+    $theme = $block->input('theme');
+    $variation = $block->input('variation');
+
+    // In the case of a variation swap for multiple we want to limit the number of items to 3 to fit design
+
+    if ($theme === 'multiple' && ($variation === 'experience-with-us' || $variation === 'make-with-us')) {
+        $block->childs = $block->childs->sortBy('sort_order')->take(3);
+    }        
 @endphp
 
-<div id="{{ str(strip_tags($header))->kebab() }}" class="m-showcase-block">
-    <div class="m-showcase-wrapper">
-        @if ($header)
-            <h3 class="showcase-header">{{ $header }}</h3>
-        @endif
-        @if ($mediaType == 'video')
-            @component('components.atoms._video')
-                @slot('video', ['src' => $video])
-                @slot('controls', true)
-                @slot('class', 'm-showcase-video')
-            @endcomponent
-        @else
-            @component('components.atoms._img')
-                @slot('image', $image)
-                @slot('settings', $imageSettings ?? '')
-                @slot('class', 'm-showcase-image')
-            @endcomponent
-        @endif
-        <div class="m-showcase-block__text-wrapper">
-            @if ($tag)
-                @component('components.atoms._title')
-                    @slot('tag', 'span')
-                    @slot('font', 'f-tag')
-                    @slot('variation', 'showcase-tag')
-                    @slot('title', $tag)
-                @endcomponent
-            @endif
-            @if ($title)
-                @component('components.atoms._title')
-                    @slot('tag', 'div')
-                    @slot('font', 'f-headline-editorial')
-                    @slot('variation', 'showcase-title')
-                    @slot('title', $title)
-                @endcomponent
+@if($theme !== 'multiple')
+    <div class="m-showcase-block {{ $theme ? 'showcase--'.$theme : '' }} {{ $variation ? 'showcase--variation-'.$variation : '' }}">    
+        <div class="m-showcase-wrapper">
+            @if ($header)
+                <h3 class="showcase-header">{{ $header }}</h3>
```

Comment:
In the RLC showcase block, if I enter a header but choose the default layout, the header is still displayed when it's not part of the design:

<img width="901" alt="Screenshot 2024-02-06 at 1 04 29 PM" src="https://github.com/art-institute-of-chicago/artic.edu/assets/1613175/b0b3cfe9-0313-4429-862e-1fa535aa073d">

CSS should hide this field in the default theme layout.